packages/common/src/hooks/chats/useCanSendMessage.ts

```diff
@@ -12,14 +12,20 @@ import { useProxySelector } from '..'
 /**
  * Returns whether or not the current user can send messages to the current chat
  */
-export const useCanSendMessage = (currentChatId?: string) => {
+export const useCanSendMessage = (
+  currentChatId?: string
+): {
+  canSendMessage: boolean
+  callToAction: ChatPermissionAction
+  // Explicitly define type as undefinable since users could be empty
+  firstOtherUser: User | undefined
+} => {
   const users = useProxySelector(
     (state) => getOtherChatUsers(state, currentChatId),
     [currentChatId]
   )
 
-  // Explicitly define type as undefinable since users could be empty
-  const firstOtherUser: User | undefined = users[0]
+  const firstOtherUser = users[0]
```

Comment:
Might be able to avoid the returntype with `const firstOtherUser = users[0] as User | undefined`